packages/server/pkg/config/config.go

```diff
@@ -10,7 +12,7 @@ import (
 
 // Config describes all configuration options
 type Config struct {
-	Database string `default:"pgsql://localhost/nebula" usage:"PostgreSQL DSN to connect to (i.e. pgsql://localhost/nebula)"`
+	Database string `default:"$DATABASE_URL" usage:"PostgreSQL DSN to connect to (i.e. postgres://localhost/nebula)"`
```

Comment:
Sorry, I completely forgot about this. You can actually set this value through `NEBULA_DATABASE` already (just like all other configuration settings). The `NEBULA_` prefix comes from `EnvPrefix` in the `Loader()` function.
Using that would be much simpler than adding a magic value. Thoughts?

Reply:
I prefer `$DATABASE_URI`, but I acknowledge this would be the path of least resistance. Works for me.